WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -266,6 +266,9 @@ def post_results
       unless comp.results
         return render html: "<div class='container'><div class='alert alert-warning'>No results</div></div>".html_safe
       end
+      unless comp.results_posted.nil?
```

Comment:
I think this is causing test failures, see https://travis-ci.org/thewca/worldcubeassociation.org/builds/519622357#L6266